src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -255,6 +358,9 @@ private Concept autoMergeConcept(Concept sourceConcept, Concept targetConcept) {
 		mergedConcept.setDefinitionStatus(winningConcept.getDefinitionStatus());
 		mergedConcept.setEffectiveTimeI(winningConcept.getEffectiveTimeI());
 		mergedConcept.setModuleId(winningConcept.getModuleId());
+		mergedConcept.setReleased(winningConcept.isReleased());
+		mergedConcept.setReleaseHash(winningConcept.getReleaseHash());
+		mergedConcept.setReleasedEffectiveTime(winningConcept.getReleasedEffectiveTime());
```

Comment:
I noticed these properties weren't copied over when I was stepping through some tests.

Reply:
Good catch. I thought we are going to improve the auto merge logic too in our discussions.